students/chanjoo/westargram/users/decorators.py

```diff
@@ -0,0 +1,30 @@
+import json
+import datetime
+
+import jwt
+from jwt.exceptions import DecodeError, ExpiredSignatureError
+from django.http import JsonResponse
+from django.contrib.auth.models import User
```

Comment:
import을 기준으로 정렬해 주세요.(wecode 컨벤션)
`from django.contrib.auth.models import User`을 import 한 이유가 있을까요? 사용하지 않으면 제거해 주세요. 
```suggestion
import jwt
from jwt.exceptions             import DecodeError, ExpiredSignatureError
from django.http                import JsonResponse
from django.contrib.auth.models import User
```